### ISSUE TITLE
About the EM/F1 score in pre-trained model

### ISSUE THREAD

**Issue by YeDeming** [allenai/document-qa#15]
Hi Christopher,

Thanks for open-sourcing your code!

When I try to use your provided pre-trained "triviaqa-web-shared-norm" model to test in the dev set. I get  the results as follow:

N Paragraphs EM     F1
1            0.5807 0.6339
2            0.6307 0.6818
3            0.6446 0.6945
4            0.6498 0.6993
5            0.6532 0.7024
6            0.6543 0.7035
7            0.6554 0.7045
8            0.6562 0.7053
9            0.6568 0.7058
10           0.6570 0.7059
11           0.6573 0.7062
12           0.6575 0.7064
13           0.6576 0.7064
14           0.6577 0.7065
15           0.6577 0.7064

How could I sum up all answers from different paragraphs to get a unique answer?  In other words, how could I get the output of EM/F1=66.37/71.32,  which is reported by the paper?

Thanks a lot for your time!
YeDeming

**Comment by chrisc36** [comment]
The last score (n_paragraphs=15) is the result using the most confidence answer across fifteen paragraphs, and is what we used for our official results.

The EM/F1=66.37/71.32 score is on the test set, on the dev you should get 66.05/70.99. To get that result run the evaluation script with a paragraph size of 800, like:

python docqa/eval/triviaqa_full_document_eval.py --n_processes 6 -t 800 -c web-dev /path/to/model

**Comment by YeDeming** [comment]
Thanks! I have got the point.
